### DISCUSSION TITLE
Weather info

### DISCUSSION THREAD

**Discussion by cmoneto** [BimmerGestalt/AAIdrive#718] [Ideas]
hello sir,  great work!
i was thinking that a weather integration could be nice. i look for it but apperntly no one has said somethig about it. 
maybe i dont get weather info on my mini since i dont have mini connect subscription. 

thanks!
CM
Mini F56 2024

**Comment by hufman** [comment]
Thank you for the compliments!
Indeed, AAIdrive does not provide a weather integration. AAIdrive can only add UI to the car by borrowing from previous BMW Android apps (Spotify, SmartThings, BMW Connected Classic), and I don't know of any previous weather apps from BMW.
Most BMWs (and my 2021 Mini, anecdotally, which also doesn't have a subscription) come with a weather app already, which is probably why it hasn't been requested from AAIdrive.

**Reply by cmoneto** [reply]
Thanks for the reply. I haven't seen the weather app on my mini. I'm going to look for it but how would the mini weather app gets the weather Info? Since I dot have a subscription... My car doesn't get internet connection.

Thanks!

**Reply by hufman** [reply]
My Mini has an LTE modem to receive remote commands, traffic info, and other data (WikiLocal and Weather are the main onboard internet apps I can think of).

**Reply by cmoneto** [reply]
Do your know I way I can get that to work? Since mini connect isn't supported on my country. I can't get a data connection for my car but Im sure it does have the hardware.

**Reply by hufman** [reply]
Unfortunately, I don't know anything about modifying the car itself. Try checking around for BMW Coding shops, which may be able to help better than I can.

**Reply by cmoneto** [reply]
Thanks!
